tycho-bundles/org.eclipse.tycho.p2.resolver.impl.test/resources/targetresolver/mavenDep.target

```diff
@@ -2,7 +2,7 @@
 <?pde version="3.8"?>
 <target name="mavenDep.targe">
 	<locations>
-		<location includeDependencyDepth="none" includeDependencyScope="compile" includeSource="false" missingManifest="ignore" type="Maven">
+		<location includeDependencyScope="compile" missingManifest="ignore" type="Maven">
```

Comment:
This removes the latest changes from m2e so maybe another reason why the test fails.